### DISCUSSION TITLE
instance_properties_json field es location

### DISCUSSION THREAD

**Discussion by mitht** [apache/skywalking#11598] [Q&A] [answered]
Where are the data for instance_properties_json and JVM arguments stored in Elasticsearch, and on which index and field?

es index:
sw_records-all       sw_records-all-20231127       - - -
sw_zipkin_span       sw_zipkin_span-20231127       - - -
sw_browser_error_log sw_browser_error_log-20231127 - - -
sw_segment           sw_segment-20231127           - - -
sw_metrics-all       sw_metrics-all-20231127  

<img width="1613" alt="image" src="https://github.com/apache/skywalking/assets/29425860/1141db2c-4f38-4042-8404-d66e9e47dca7">

**Comment by wu-sheng** [comment, marked as answer]
In the `sw_metrics-al`, and table_name is `instance_traffic`.

**Reply by mitht** [reply]
Thank you so much, I found it
